core-bundle/src/Resources/contao/library/Contao/Controller.php

```diff
@@ -668,32 +668,6 @@ public static function getForm($varId, $strColumn='main', $blnModule=false)
 		return $strBuffer;
 	}
 
-	/**
-	 * Return the languages for the TinyMCE spellchecker
-	 *
-	 * @return string The TinyMCE spellchecker language string
-	 */
-	protected function getSpellcheckerString()
-	{
-		System::loadLanguageFile('languages');
-
-		$return = array();
-		$langs = Folder::scan(__DIR__ . '/../../languages');
-		array_unshift($langs, $GLOBALS['TL_LANGUAGE']);
-
-		foreach ($langs as $lang)
-		{
-			$lang = substr($lang, 0, 2);
-
-			if (isset($GLOBALS['TL_LANG']['LNG'][$lang]))
-			{
-				$return[$lang] = $GLOBALS['TL_LANG']['LNG'][$lang] . '=' . $lang;
-			}
-		}
-
-		return '+' . implode(',', array_unique($return));
-	}
```

Comment:
@contao/developers Can I remove this method here? It is protected and not used anywhere.
I will add a deprecation for Contao 4.13 in a seperate PR.

Reply:
See #4567